src/dev/konnector.md

```diff
@@ -2,110 +2,376 @@
+    },
+    fileurl: {
+      sel: 'img',
+      attr: 'src',
+      parse: src => `${baseUrl}/${src}`
+    },
+    filename: {
+      sel: 'h3 a',
+      attr: 'title',
+      parse: title => `${title}.jpg`
+    }
+  },
+  'article'
+)
+```
+
+This code will loop on `<article />` and for each item will create a JSON object with the selector `sel` and the value of attribute `attr` if specified, otherwise it takes the value of the child node, this value can be edited with the `parse` function.
+Here is a sample for the following markup from http://books.toscrape.com:
+
+```html
+<article class="product_pod">
+  <div class="image_container">
+    <a href="catalogue/a-light-in-the-attic_1000/index.html">
+      <img src="media/cache/2c/da/2cdad67c44b002e7ead0cc35693c0e8b.jpg" alt="A Light in the Attic" class="thumbnail">
+    </a>
+  </div>
+  <p class="star-rating Three">
+    <i class="icon-star"></i>
+    <i class="icon-star"></i>
+    <i class="icon-star"></i>
+    <i class="icon-star"></i>
+    <i class="icon-star"></i>
+  </p>
+  <h3>
+    <a href="catalogue/a-light-in-the-attic_1000/index.html" title="A Light in the Attic">A Light in the ...</a>
+  </h3>
+  <div class="product_price">
+    <p class="price_color">£51.77</p>
+    <p class="instock availability">
+      <i class="icon-ok"></i>
+      In stock
+    </p>
+    <form>
+      <button type="submit" class="btn btn-primary btn-block" data-loading-text="Adding...">Add to basket</button>
+    </form>
+  </div>
+</article>
+```
+
+And we will get the following JSON object:
+
+```json
+{
+  "title": "A Light in the Attic",
+  "amount": 51.77,
+  "url": "http://books.toscrape.com/catalogue/a-light-in-the-attic_1000/index.html",
+  "fileurl": "http://books.toscrape.com/media/cache/2c/da/2cdad67c44b002e7ead0cc35693c0e8b.jpg",
+  "filename": "A Light in the Attic.jpg"
+}
+```
+
+The code sample includes some other function to manipulate the result object, but we have the idea.
+Once we build a correct object, we can save it to Cozy Stack.
+
+#### Save data to Cozy Stack
+
+In the example we use some built-in function to save a bill to the Cozy Stack.
+But there is a bunch of functions available depending on what you want:
+
+- [`addData`](https://github.com/konnectors/libs/blob/master/packages/cozy-konnector-libs/docs/api.md#adddata)
+- [`filterData`](https://github.com/konnectors/libs/blob/master/packages/cozy-konnector-libs/docs/api.md#module_filterData)
+- [`saveBills`](https://github.com/konnectors/libs/blob/master/packages/cozy-konnector-libs/docs/api.md#module_saveBills)
+- [`saveFiles`](https://github.com/konnectors/libs/blob/master/packages/cozy-konnector-libs/docs/api.md#module_saveFiles)
+- and so on…
+
+We can find more information in the [libs repository](https://github.com/konnectors/libs).
+
+__Now we pass on every steps, it is time to test the connector with `yarn standalone`.__
+We will see in the following how to connect it effectively to a Cozy Stack.
```

Comment:
the following… part? section?

Reply:
https://www.deepl.com/translator#fr/en/Nous%20verrons%20dans%20ce%20qui%20suit%20comment%20le%20connecter%20efficacement%20%C3%A0%20un%20Cozy%20Stack.